CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 # Default Owners
-* @containers/podman-desktop-reviewers @benoitf
+* @podman-desktop/reviewers @benoitf
 
 # website
-website/** @containers/podman-desktop-reviewers @slemeur @cdrage @benoitf @Firewall
-website/docs/** @containers/podman-desktop-reviewers @shipsing @slemeur @cdrage @benoitf @Firewall
+website/** @podman-desktop/reviewers @slemeur @cdrage @benoitf @Firewall
+website/docs/** @podman-desktop/reviewers @shipsing @slemeur @cdrage @benoitf @Firewall
```

Comment:
error on shipsing ?

Reply:
she needs to accept the invitation, once it's done the error will be fixed 👍